partner_store/models/res_partner.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import logging
+
+from odoo import fields, models
+
+_logger = logging.getLogger(__name__)
```

Comment:
logging is not used, so this and import should be removed. How did this get through pre-commit?